components/com_content/src/Controller/ArticleController.php

```diff
@@ -403,6 +404,39 @@ public function save($key = null, $urlVar = 'a_id')
 		return $result;
 	}
 
+	/**
+	 * Method to save article's title
+	 *
+	 * @return void
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public function saveTitle()
+	{
+		$this->checkToken();
+
+		$articleId = $this->input->getInt('a_id');
+		$articleTitle = $this->input->getString('a_title');
+
+		if (!$this->allowEdit(['id' => $articleId]))
+		{
+			echo new JsonResponse(['saved' => false, 'permission' => false]);
+		}
+		else
+		{
+			$model = $this->getModel($this->context);
```

Comment:
Would this always get the right model?
Another question, can I pass a string here since I know what model to get every time?

Reply:
@rs4231199 Check the code with XDebug and you will see how this function works. If it always gets the right model, that should be the case if you are in the context of articles.

> Another question, can I pass a string here since I know what model to get every time?

You are passing a string, check the content of `$this->context`

Reply:
> You are passing a string, check the content of `$this->context`

The value of this property is "article". I meant passing "article" instead of `$this->context`.
But `$this->context` will get the right value. So, I will leave it at that.